src/components/dropdown/Dropdown.tsx

```diff
@@ -262,6 +257,14 @@ export class Dropdown<T = LabelValue> extends React.Component<
           menuShouldBlockScroll={menuShouldBlockScroll}
           menuShouldScrollIntoView={menuShouldScrollIntoView}
         />
+        <LabelTooltipDecorator
```

Comment:
is it normal we move this to the bottom ?

Reply:
Yes... for keyboard navigation (but kept visually at the top with `order: -1`)